ern-runner-gen-ios/src/IosRunnerGenerator.ts

```diff
@@ -21,16 +21,26 @@ export default class IosRunerGenerator implements RunnerGenerator {
     const mustacheView = this.createMustacheView({ config })
 
     shell.cp('-R', path.join(runnerHullPath, '*'), config.outDir)
+
     const filesToMustache = [
-      path.join(config.outDir, 'ErnRunner/RunnerConfig.m'),
-      path.join(config.outDir, 'ErnRunner.xcodeproj/project.pbxproj'),
+      'ErnRunner/RunnerConfig.m',
+      'ErnRunner.xcodeproj/project.pbxproj',
     ]
 
+    if ((mustacheView as any).RN_VERSION_LT_61) {
+      // Delete ErnRunner.xcworkspace directory as it is only needed for RN61+
+      shell.rm('-rf', path.join(config.outDir, 'ErnRunner.xcworkspace'))
+    } else {
+      // Otherwise keep it and include the xcworkspaceedata file for mustached
+      // processing as it contains some mustache templace placeholders
```

Comment:
Minor typos "xcworkspacedata"; also "mustache" instead of "mustached"?
templace -> template